azuread/table_azuread_user.go

```diff
@@ -53,6 +53,7 @@ func tableAzureAdUser() *plugin.Table {
 			{Name: "filter", Type: proto.ColumnType_STRING, Transform: transform.FromQual("filter"), Description: "Odata query to search for resources."},
 
 			// Other fields
+			{Name: "on_premises_immutable_id", Type: proto.ColumnType_STRING, Description: "Used to associate an on-premises Active Directory user account with their Azure AD user object."},
```

Comment:
@maceip Could you please sort the columns in `Other fields` section. `on_premises_immutable_id` column should be after `mail_nickname` column.